unstructured/file_utils/filetype.py

```diff
@@ -105,102 +109,202 @@ def __lt__(self, other):
     ".rtf": FileType.RTF,
 }
 
+def _detect_filetype_from_extension(extension: str) -> Optional[FileType]:
+    """Detect the file type based on the file extension."""
+    if extension in ['.txt', '.text']:
+        return FileType.TEXT
+    elif extension in ['.pdf']:
+        return FileType.PDF
+    elif extension in ['.jpg', '.jpeg', '.png', '.gif']:
+        return FileType.IMAGE
+    # Add more supported file types and their extensions here.
+    return None
 
-def detect_filetype(
+def detect_filetype(    
```

Comment:
Did you mean to add this whitespace at the end here? You may be able to resolve this with `make tidy`.

Reply:
ok will do

Reply:
There is no unstructured/__version__.py file so should i make it and bump the version in that file?

Reply:
This one here's the version file https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/__version__.py

Reply:
I tried running the make tidy command but i think that it is not compatible with windows so should i just pip install black and format the file?

Reply:
Yeah if you just run `black` on it that should do the trick